polytope/solvers.py

```diff
@@ -50,9 +50,17 @@
 except ImportError:
     logger.info('MOSEK solver not found.')
 
+try:
+    import gurobipy as gp
+    installed_solvers.add('gurobi')
+except ImportError:
+    logger.info('GUROBI solver not found')
+
 
 # choose default from installed choices
-if 'glpk' in installed_solvers:
+if 'gurobi' in installed_solvers:
+    default_solver = 'gurobi'
```

Comment:
I lean towards not making `gurobi` a default solver, because `gurobi` and `gurobipy` are proprietary, closed-source software: https://www.gurobi.com/downloads/end-user-license-agreement-academic/, and: https://pypi.org/project/gurobipy/.

Reply:
I agree with not making `gurobi` a default solver. This is not a vote against Gurobi, but rather, automatically using a restrictive-license-encumbered dependency is an anti-pattern of open source code. Users should have to explicitly request it.

Perhaps there is a more convenient way for users to do this, e.g., an environment variable.